code-challenges.js

```diff
@@ -47,3 +97,33 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+
+//create a funtion that takes in a set of any two numbs 
+//subtracts the smaller number
+//from the larger
+//console logs the expexted output
+
+
+
+if(number1 < number2){ 
+console.log(number2 - number1)
+}
+  else if(number1 > number2){
+console.log(number1 - number2)
+ }
+
+else{
+  console.log("the numbers are not equal")
+}
```

Comment:
Logic looks really good, I like that you thought about the instance of what if the numbers are equal.  Just make sure the console log is stating that the numbers _are_ equal.  Your pseudocode mentions creating a function which would make this conditional more dynamic and reusable.  Just something to consider if you find yourself writing the same thing to match a different set of variables.